dashboard/settings.py

```diff
@@ -45,6 +45,11 @@
 LOGOUT_URL = '/accounts/logout'
 HELP_URL = ENV.get("HELP_URL", "https://sites.google.com/umich.edu/my-learning-analytics-help/home")
 
+URL_VIEW_RESOURCES_ACCESSED = ENV.get("URL_VIEW_RESOURCES_ACCESSED", "https://its.umich.edu/academics-research/teaching-learning/my-learning-analytics/support/resources-accessed")
```

Comment:
I don't think we should have defaults in the code for umich resources, these should also be documented in the env_sample file: https://github.com/tl-its-umich-edu/my-learning-analytics/blob/master/config/env_sample.json

We can suggest these as help documentation that may want to be customized.

Reply:
Talked to @jennlove-um and going to leave this as is and treat handling lack of help in a future issue.

Reply:
These should be suffix URL fragments that are added to a base help URL.

Reply:
I didn't see any user benefit in requiring urls to be relatively located so i removed that to permit flexibility.
The previous design required the help pages to all have specific urls based off a prefix.  Which I suppose you are advocating.  I thought this was inflexible.  There are benefits to both I guess.